spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/event/asset/AssetDeleted.java

```diff
@@ -9,25 +9,25 @@
  *
  *  Contributors:
  *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *       Fraunhofer Institute for Software and Systems Engineering - expending Event classes
  *
  */
 
 package org.eclipse.dataspaceconnector.spi.event.asset;
 
 import org.eclipse.dataspaceconnector.spi.event.Event;
-import org.eclipse.dataspaceconnector.spi.event.EventPayload;
 
 import java.util.Objects;
 
 /**
  * Describe an Asset deletion, after this has emitted, the Asset represented by the id won't be available anymore.
  */
-public class AssetDeleted extends Event<AssetDeleted.Payload> {
+public class AssetDeleted extends AssetEvent {
 
     private AssetDeleted() {
     }
 
-    public static class Builder extends Event.Builder<AssetDeleted, Payload, Builder> {
+    public static class Builder extends Event.Builder<AssetDeleted, AssetEvent.Payload, Builder> {
```

Comment:
Same as above, and at a this point I would move the builder logic that permits to add an `assetId` on the `AssetEvent` superclass